spec/round_spec.rb

```diff
@@ -0,0 +1,182 @@
+        round = Round.new(deck)
+        new_turn = round.take_turn("Juneau")
+        next_turn = round.take_turn("Venus")
+        expect(round.turns.last.feedback).to eq("Incorrect.")
+    end
+    
+    it 'has a number correct' do
+        card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+        card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+        card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+        deck = Deck.new([card_1, card_2, card_3])
+        round = Round.new(deck)
+        new_turn = round.take_turn("Juneau")
+        next_turn = round.take_turn("Venus")
+        expect(round.number_correct).to eq(1)
+    end
+
+    it 'has a number correct by category' do
+        card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+        card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+        card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+        deck = Deck.new([card_1, card_2, card_3])
+        round = Round.new(deck)
+        new_turn = round.take_turn("Juneau")
+        next_turn = round.take_turn("Venus")
+        expect(round.number_correct_by_category(:Geography)).to eq(1)
+        expect(round.number_correct_by_category(:STEM)).to eq(0)
+    end
+
+    it 'has a percentage correct for the round' do
+        card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+        card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+        card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+        deck = Deck.new([card_1, card_2, card_3])
+        round = Round.new(deck)
+        new_turn = round.take_turn("Juneau")
+        next_turn = round.take_turn("Venus")
+        expect(round.percent_correct).to eq(50.0)
+    end
+
+    it 'has a percent correct by category' do
+        card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+        card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+        card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+        deck = Deck.new([card_1, card_2, card_3])
+        round = Round.new(deck)
+        new_turn = round.take_turn("Juneau")
+        next_turn = round.take_turn("Venus")
+        following_turn = round.take_turn("North north west")
+        expect(round.percent_correct_by_category(:Geography)).to eq(100.0)
+        
```

Comment:
If we add another assertion here to make sure it's working correctly for percentages less than 100, we will see that this method isn't working quite as intended.
`expect(round.percent_correct_by_category(:STEM)).to eq(50.0)`